src/haddock/clis/cli_cp.py

```diff
@@ -147,10 +148,11 @@ def main(run_dir, modules, output):
     copy_renum_step_folders(run_dir, outdir, selected_steps)
 
     # copy data folders
```

Comment:
```suggestion
# copy data folders
# `data_steps` are selected to avoid FileNotFoundError because some steps do not have a `data` folder.
# See https://github.com/haddocking/haddock3/issues/559
```

Reply:
thanks for the review, I added the comment!